packages/trader/src/App/Components/Elements/__tests__/market-countdown-timer.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types';
+import { ActiveSymbols, TradingTimesResponse } from '@deriv/api-types';
+import MarketCountdownTimer from '../market-countdown-timer';
+import TraderProviders from '../../../../trader-providers';
+
+const mock_default_props = {
+    is_main_page: false,
+    setIsTimerLoading: jest.fn(),
+    onMarketOpen: jest.fn(),
+    symbol: 'WLDAUD',
+};
+
+const default_mock_store = {
+    modules: {
+        trade: {
+            active_symbols: [
+                {
+                    allow_forward_starting: 0,
+                    display_name: 'AUD Basket',
+                    display_order: 26,
+                    exchange_is_open: 1,
+                    is_trading_suspended: 0,
+                    market: 'synthetic_index',
+                    market_display_name: 'Derived',
+                    pip: 0.001,
+                    subgroup: 'baskets',
+                    subgroup_display_name: 'Baskets',
+                    submarket: 'forex_basket',
+                    submarket_display_name: 'Forex Basket',
+                    symbol: 'WLDAUD',
+                    symbol_type: 'forex_basket',
+                },
+            ] as ActiveSymbols,
+        },
+    },
+};
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        tradingTimes: () => ({
+            api_initial_load_error: false,
+            trading_times: {
+                markets: [
+                    {
+                        submarkets: [
+                            {
+                                symbols: [
+                                    { symbol: 'WLDAUD', times: { open: ['08:00:00.153'], close: ['22:00:00.123'] } },
+                                ],
+                            },
+                        ],
+                    },
+                ],
+            } as unknown as NonNullable<DeepRequired<TradingTimesResponse['trading_times']>>,
+        }),
+    },
+}));
+
+describe('<MarketCountdownTimer />', () => {
+    Date.now = jest.fn(() => new Date(Date.UTC(2017, 1, 14)).valueOf());
```

Comment:
@kate-deriv  nit-pick: i would suggest putting the resulting value, it seems to be shorter :)  1487030400000 is the result of new Date(Date.UTC(2017, 1, 14)).valueOf().

```suggestion
    Date.now = jest.fn(() => 1487030400000);
```